.github/workflows/ci.yml

```diff
@@ -60,6 +70,76 @@ jobs:
         run: |
           cmake --build build --target check --config "${{ matrix.cfg.type }}" -j 2
 
+  # HACK to enable/disable the max CI based on presence/absence of secret. See also:
+  # https://github.com/actions/runner/issues/1138
+  max-secrets:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    name: Check for Max SDK
+    outputs:
+      HAS_MACHINE_USER_TOKEN: ${{ steps.check.outputs.MACHINE_USER_TOKEN }}
+    steps:
+      - name: Check for Secret
+        run: >
+          echo "::set-output name=MACHINE_USER_TOKEN::${{ env.MACHINE_USER_TOKEN != '' }}";
+        id: check
+        env:
+          MACHINE_USER_TOKEN: ${{ secrets.MACHINE_USER_REPO_READ }}
+
+  max:
+    # Can only run if we have a token for our super seekrit Max SDK repo. Sad.
+    needs: ["max-secrets"]
+    if: needs.max-secrets.outputs.HAS_MACHINE_USER_TOKEN == 'true'
+
+    runs-on: windows-latest
+    name: maxplugin-${{ matrix.cfg.str }}
+    strategy:
+      matrix:
+        cfg:
+          - { sdk-version: 2008, generator: Visual Studio 16 2019, arch: Win32, str: 2008-windows-x86 }
```

Comment:
A comment here indicating that Max 7 and 2020 are tested as part of the CI workflows would be helpful for some future person looking here and being confused about missing versions 😛